File: lib/typescript/botbuilder-skills/src/Models/skillDialogOptions.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { ISkillManifest } from './skillManifest';

export interface ISkillDialogOptions {
   skillManifest: ISkillManifest | undefined;
   parameters: Map<string, Object>;
}

File: sdk/csharp/libraries/microsoft.bot.builder.solutions.skills/Models/SkillDialogOptions.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using System.Collections.Generic;
using Microsoft.Bot.Builder.Skills.Models.Manifest;

namespace Microsoft.Bot.Builder.Skills.Models
{
    public class SkillDialogOptions
    {
        public SkillManifest SkillManifest { get; set; }

        public Dictionary<string, object> Parameters { get; set; } = new Dictionary<string, object>();
    }
}